cmd/uninstall.go

```diff
@@ -35,10 +34,6 @@ var uninstallCommand = &cobra.Command{
 			return fmt.Errorf("pre-uninstall install script failed: %s", err.Error())
 		}
 
-		if !prompter.YN(fmt.Sprintf("This will permanently delete '%s'. Are you sure?", path), false) {
```

Comment:
can we make it `--force` as we have in [install](https://github.com/whalebrew/whalebrew/blob/master/cmd/install.go#L24) ?